ens/main.py

```diff
@@ -79,15 +81,11 @@
     )
 
 
-class ENS:
```

Comment:
Thanks for the work @dbfreem! I haven't taken a look at any specifics yet but I would ideally like for this PR to be when we start to separate larger async and sync classes based on a similar structure to [this comment](https://github.com/ethereum/web3.py/pull/2441#pullrequestreview-966920702). Thoughts @kclowes?

I'd be happy to take that on if you'd like @dbfreem since you've thrown this all together already. It's late today but I plan on taking a closer look at specifics here this week. Just wanted to get the conversation started again on the sync + async organization moving forward.

Reply:
@fselmo I was thinking about doing that in the first place but then thought it might make the PR too big. I went ahead and reorganized it take a look and let me know what you think.